serving/src/main/java/feast/serving/util/SpecUtil.java

```diff
@@ -0,0 +1,11 @@
+package feast.serving.util;
+
+public class SpecUtil {
```

Comment:
`SpecUtil` -> `FeatureReference`
`generateFeastId` -> `generateStringFeatureReference`

Reply:
FeatureReference is an actual object though

Reply:
I was wondering if this could be the start of a facade, but we only implement the static method. Alternatively `FeatureRefUtil`

Reply:
And its used for both features and featuresets

Reply:
Im not a huge fan of introducing a facade just to generate this id

Reply:
`FeatureRefUtil` then?